tensorflow/python/ops/nn_ops.py

```diff
@@ -1294,7 +1294,7 @@ def conv3d_transpose(value,
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
       See the @{tf.nn.convolution$comment here}
     data_format: A string, either `'NDHWC'` or `'NCDHW`' specifying the layout
-      of the input and output tensors. Defaults to `'NDHWC'`.
```

Comment:
Either we should keep this docstring or we should modify the method signatures of all these ops to have a default `data_format`, which is a string instead of NoneType. And probably introduce some flexibility with `channels_first`, `channels_last` and all these `NHWC` formats. Currently passed `NHWC` to `tf.layers` constructors throws an error which I personally found uncomfortable.

Your opinions @fchollet @drpngx ?